linera-rpc/src/grpc/client.rs

```diff
@@ -65,6 +65,23 @@ impl GrpcClient {
         })
     }
 
+    pub fn from_address_channel(
```

Comment:
Shouldn't this be the constructor `new` ?

Reply:
Yes, done.